spec/controllers/api/auth/registrations/create_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Sign Up user via POST', type: :request do  
+  it "should create User when given valid email, gender, password and password confirmation" do
+    expect {
+      post '/api/auth/users', params: { 
+        user: { 
+          email: 'test@mail.com', 
+          password: 'testing', 
+          password_confirmation: 'testing', 
+          gender: 'male' 
+        } 
+      }
```

Comment:
You can define a `subject` in this file so it can be used in tests below. Try to no repeat yourself